templates/lxd/index.html

```diff
@@ -4,337 +4,437 @@
 
 {% block title %}Canonical LXD{% endblock %}
 
-{% block meta_description %}Low-touch virtual infrastructure with LXD at any scale. Built on top of Linux containers,
-LXD offers a unified user experience in managing system containers, virtual machines and clusters. Easily configurable,
-flexible and secure way to run your workloads.{% endblock %}
+{% block meta_description %}
+  Low-touch virtual infrastructure with LXD at any scale. Built on top of Linux containers,
+  LXD offers a unified user experience in managing system containers, virtual machines and clusters. Easily configurable,
+  flexible and secure way to run your workloads.
+{% endblock %}
 
-{% block body_class %}is-paper{% endblock body_class %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
 
-<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
-  <div class="p-section">
-    <div class="row--25-75">
-      <div class="col p-section">
-        <div class="p-section--shallow">
-          <h1 class="u-no-margin--bottom">Run system containers with LXD</h1>
-          <h2>Fast, dense, and secure container and VM management at any scale</h2>
+  <section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row--25-75">
+        <div class="col p-section">
+          <div class="p-section--shallow">
+            <h1 class="u-no-margin--bottom">Lightweight open source virtualisation with LXD</h1>
+            <h2>The comprehensive platform for open source virtualization with a cloud-like experience.</h2>
+          </div>
+          <div class="p-section--shallow">
+            <p>
+              LXD is an open-source solution for managing virtual machines and system containers. It provides a secure and scalable environment with minimal overhead. Manage your workloads with ease and configure them to suit your use case via a user-friendly web interface.
+            </p>
+          </div>
+          <a class="p-button--positive" href="/lxd/install">Install now</a>
         </div>
-        <div class="p-section--shallow">
-          <p>LXD provides a unified user experience for managing system containers and virtual machines. For more
-            demanding workloads, LXD can be set up in a cluster environment to run containers, VMs, or a combination of
-            the two on a set of machines. LXD has direct hardware access, minimising overhead and matching the density
-            and efficiency of containers.</p>
+      </div>
+      <div class="row u-hide--medium u-hide--small">
+        <div class="col-9 col-start-large-4">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/410648c5-Artboard%2070@2x.png",
+          alt="",
+          width="496",
+          height="375",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+          }}
```

Comment:
Can you please change the image tags to this format? djlint doesn't do this automatically but we want to make it consistent in this format
```suggestion
          {{ image(url="https://assets.ubuntu.com/v1/410648c5-Artboard%2070@2x.png",
                          alt="",
                          width="496",
                          height="375",
                          hi_def=True,
                          loading="auto") | safe
          }}
```